### PR TITLE
Issue 7786

### PR DESCRIPTION
**Pull request by panditsavitags**
#### Summary
Migrate delete_emoji_modal.jsx to pure and to use redux

#### Ticket Link
[issue_7786](https://github.com/mattermost/mattermost-server/issues/7786)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has server changes (please link)
- [x] Has redux changes (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/emoji/components/delete_emoji_modal.jsx [hunk 1]
```diff
@@ -8,6 +8,11 @@ import {FormattedMessage} from 'react-intl';
 import DeleteModalTrigger from '../../delete_modal_trigger.jsx';
```
Comment (saturninoabril) [inline components/emoji/components/delete_emoji_modal.jsx]: Please change to ``import DeleteModalTrigger from 'components/delete_modal_trigger.jsx';``

Comment (panditsavitags) [reply components/emoji/components/delete_emoji_modal.jsx]: @saturninoabril , Actually, DeleteModalTrigger test casses cover the tests for the childClasses extending it.

Comment (panditsavitags) [reply components/emoji/components/delete_emoji_modal.jsx]: @saturninoabril , i will take care of the the other changes mentioned by you. Thanks

**Comment by panditsavitags** [comment]
@saturninoabril , Actually the parent

**Comment by panditsavitags** [comment]
@saturninoabril , DeleteEmoji extends DeleteModalTrigger and has no new rendering apart from specifying the text messages, so there will not be any snap file for this right? So only testing required is for testing the text messages returned ? Please correct me if my understanding is wrong.

**Pull request closed without merging**